From 0bf65aed180ca8246f92a443e58b0b8b5008ae4f Mon Sep 17 00:00:00 2001
From: Katsuya Matsubara <matsu@igel.co.jp>
Date: Thu, 31 May 2012 15:19:49 +0900
Subject: [PATCH 29/80] rcarvin: initialize the MTC register for memory transfer

According to the hardware manual, the MTC register must be initialized
with 0x0a090008.

Signed-off-by: Katsuya Matsubara <matsu@igel.co.jp>
---
 drivers/media/video/rcarvin.c |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/drivers/media/video/rcarvin.c b/drivers/media/video/rcarvin.c
index f612931..6ed45ce 100644
--- a/drivers/media/video/rcarvin.c
+++ b/drivers/media/video/rcarvin.c
@@ -399,6 +399,9 @@ static int rcar_vin_capture(struct rcar_vin_dev *pcdev)
 	/* disable interrupt */
 	vin_write(pcdev, V0IE, 0x00000000);
 
+	/* set priority for memory transfer */
+	vin_write(pcdev, V0MTC, 0x0a090008);
+
 	switch (pcdev->field) {
 	case V4L2_FIELD_TOP:
 		field = VIN_VNMC_IM_ODD;
-- 
1.7.0.4

